spec/models/game_spec.rb

```diff
@@ -43,4 +43,33 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  describe "Populate Board" do
+    it "should have correct number of pieces" do
+      game = FactoryGirl.create(:game)
+      expect(game.pieces.count).to eq 32
+    end
+
+    it "Pawn is in correct position" do
+      game = FactoryGirl.create(:game)
+      expect(game.pieces.first.type).to eq("Pawn")
```

Comment:
The order that the board gets populated in doesn't actually matter, so I wouldn't create tests that rely on a specific ordering with syntax like `pieces.first`